sanity/doctors/ParcelDoctor.js

```diff
@@ -89,16 +89,21 @@ export class ParcelDiagnosis extends Diagnosis {
   }
 
   async prepare() {
-    const deletes = this.faultyParcels.map(parcel =>
-      BlockchainEvent.deleteByArgs('assetId', parcel.token_id)
-    )
+    let deletes = []
+    for (const parcel of this.faultyParcels) {
+      deletes = deletes.concat([
+        BlockchainEvent.deleteByArgs('assetId', parcel.token_id),
```

Comment:
Why we dont check also the event name? Is there any way to delete something undesired?